enterprise-suite/tests/smoke_prometheus

```diff
@@ -250,6 +252,20 @@ busy_wait app_data_via_service
 
 prom_has_data "$app_instances_via_service"
 
+
+monitor_exists() {
+  results=$( prom_query "model{name=\"$1\"}" | jq '.data.result | length' )
+  ! [[ results -eq 0 ]]
+}
+
+# kubernetes-cadvisor metrics should have an es_monitor_type label.
+curl -XPOST -H "Content-Type: application/json" -H 'Author-Name: Me' -H 'Author-Email: me@lightbend.com' -H 'Message: testing!' \
+    "$ES_MONITOR_API/monitors/es-test/es-monitor-type-test" \
+    --data '{"model":"threshold","parameters":{"comparator":"\u003e=","confidence":"1","description":"cpu on {{ $labels.instance }} over 99% busy","meta":{},"metric":"container_cpu_load_average_10s{es_workload=\"es-test\"}","monitorType":"es-test","name":"es-monitor-type-test","severity":"warning","summary":"cpu saturated","threshold":"99","window":"5m"}}'
+
+busy_wait monitor_exists es-monitor-type-test
```

Comment:
This is not enough to guarantee that the data has been relabeled yet. It is indeed bad if this does not happen, and we cannot expect data changes until it happens, but we should busy_wait on the data being tested even so.